File: BlueMS/src/main/java/com/st/BlueMS/demos/PianoDemo/PianoKey.java
package com.st.BlueMS.demos.PianoDemo;

import android.graphics.RectF;

public class PianoKey {

    public int sound;
    public RectF rect;
    public boolean pressed;
    public boolean white;

    public PianoKey(RectF rect, int sound, boolean white) {
        this.sound = sound;
        this.rect = rect;
        this.pressed = false;
        this.white = white;
    }
}

File: st_piano/src/main/java/com/st/piano/PianoKey.kt
/*
 * Copyright (c) 2022(-0001) STMicroelectronics.
 * All rights reserved.
 * This software is licensed under terms that can be found in the LICENSE file in
 * the root directory of this software component.
 * If no LICENSE file comes with this software, it is provided AS-IS.
 */
package com.st.piano

import android.graphics.RectF


class PianoKey(var rect: RectF, var sound: Int, var white: Boolean) {
    var pressed = false
}